packages/buckets/src/buckets.spec.ts

```diff
@@ -438,9 +452,12 @@ describe('Buckets...', function () {
       await bobBuckets.getToken(bob)
       bobBuckets.withThread(aliceThread)
 
+      const { root } = await bobBuckets.listPath(rootKey, '')
       try {
+        // Note: Callers can also specify chunk size directly using highWaterMark:
+        // const stream = fs.createReadStream(path.join(pth, 'file2.jpg'), { highWaterMark: 32768 })
```

Comment:
Updating test with a little reminder for the future. Though it shouldn't be required.